keps/NNNN-kep-template/README.md

```diff
@@ -308,31 +308,43 @@ Integration tests are contained in k8s.io/kubernetes/test/integration.
 Integration tests allow control of the configuration parameters used to start the binaries under test.
 This is different from e2e tests which do not allow configuration of parameters.
 Doing this allows testing non-default options and multiple different and potentially conflicting command line options.
+For more details, see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/testing-strategy.md
```

Comment:
> Let's add a reference to the testing strategy doc in git.k8s.io/kubernetes/community ?

Added here because it already talked about integration and e2e testing.